src/AxelarGateway.sol

```diff
@@ -206,6 +206,26 @@ abstract contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
         return getBool(_getIsCommandExecutedKey(commandId));
     }
 
+    /// @dev Returns the current `adminEpoch`.
+    function getAdminEpoch() external view override returns (uint256) {
+        return _adminEpoch();
+    }
+
+    /// @dev Returns the admin threshold for a given `adminEpoch`.
+    function getAdminThreshold(uint256 adminEpoch) external view override returns (uint256) {
```

Comment:
```suggestion
    function adminThreshold(uint256 adminEpoch) external view override returns (uint256) {
```
Since the other getter is call `admins()` instead of `getAdmins()`, also better considering solidity's practice.